Makefile

```diff
@@ -87,6 +87,9 @@ GAIA_OPTIMIZE?=0
 GAIA_DEV_PIXELS_PER_PX?=1
 NGA_SERVICE_WORKERS?=0
 
+# For Data Sync
+FIREFOX_SYNC?=0
+
```

Comment:
Please write down the actual build command on the bug and/or any Data Sync document, so people who want to try it out could find a correct way to enable it.